public/index.html

```diff
@@ -197,30 +205,35 @@ <h3 class="solution__section__hdr">She checks up on you.</h3>
           <p class="screen-reader">An example text conversation between Florence and a patient</p>
           <ol class="bubble">
             <li class="bubble__florence">It’s been three days since you started taking Lisinopril. How’re you feeling?</li>
-            <li class="bubble__patient">a little weak and lightheaded</li>
+            <div class="bubble__wrapper">
+              <li class="bubble__patient">a little weak and lightheaded</li>
+            </div>
             <li class="bubble__florence">Feeling lightheaded is a common side effect of Lisinopril. But you should let Dr. Ruth Parker know as soon as possible</li>
-            <li class="bubble__florence-tail">Here’s her number: <a class="bubble__number" href="">(919) 545-2020</a></li>
+            <li class="bubble__florence bubble__florence--tail">Here’s her number: <a class="bubble__number" href="">(919) 545-2020</a></li>
```

Comment:
your BEM style seems to be going a bit off.

`bubble__florence` implies that `florence` is a child of `bubble`

And `bubble__florence--tail` implies that `florence` is a child of `bubble` and this version of `florence` has a tail

`__` signifies a child relationship, `--` signifies a variant

Reply:
> And bubble__florence--tail implies that florence is a child of bubble and this version of florence has a tail

Right. This is exactly what I meant. Is that wrong?